### ISSUE TITLE
Pose Optimization Code

### ISSUE THREAD

**Issue by OumGu** [Felix-Petersen/gendr#1]
Hello, thanks for the camera pose optimization code.

How can I improve the tutorial to optimize a 6D camera position effectively ? I am attempting to refine the camera's pose using initial values of 3 Euler angles and the x,y, and z coordinates. However, this task is challenging, and I'm facing difficulties achieving convergence.

Could you please offer any suggestions for improving the optimization process, such as selecting appropriate hyperparameters like the learning rate and the differentiable renderer parameters?

Thank you in advance.

**Comment by Felix-Petersen** [comment]
Hi, when optimizing location, it's generally advisable to (implicitly) specify for the values, e.g., via a scaled tanh transform. Generally, sigma should be larger when optimizing locations to achieve an overlap between the smoothed out differentiable rendering and the object in the reference image. As another aspect, do you provide alpha channel information in the reference image? This is quite important for good optimization performance. 

If you could share your code / snippets / gifs of the animation, I may be able to give you further insights.

Best regards,
Felix
